utils/tagged_integer.hh

```diff
@@ -84,6 +84,15 @@ using tagged_integer = tagged_tagged_integer<struct final, Tag, ValueType>;
 
 } // namespace utils
 
+template <typename Final, typename Tag, std::integral ValueType>
+struct fmt::formatter<utils::tagged_tagged_integer<Final, Tag, ValueType>> {
+    constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+    auto format(const utils::tagged_tagged_integer<Final, Tag, ValueType>& x,
+		fmt::format_context& ctx) const -> decltype(ctx.out()) {
```

Comment:
```suggestion
    auto format(const utils::tagged_tagged_integer<Final, Tag, ValueType>& x,
		fmt::format_context& ctx) const {
```
could dispense with type declaration of the return value, if we don't put the implantation in another file.

Reply:
Fixed